src/utils/generateColors.js

```diff
@@ -0,0 +1,29 @@
+import { BRAG, CONFESS, FRONT_END, BACK_END, DEV_OPS } from '../constants/buttonVariants';
```

Comment:
Make that code more abstract and connect with generateTheme.

Reply:
Use colour variants instead of buttonVariants, and use the same in generateTheme

Reply:
`FRONT_END_VARIANT`